bin/reth/src/chain/import.rs

```diff
@@ -156,9 +157,11 @@ impl ImportCommand {
                 .set(SenderRecoveryStage {
                     commit_threshold: config.stages.sender_recovery.commit_threshold,
                 })
-                .set(ExecutionStage {
-                    chain_spec: self.chain.clone(),
-                    commit_threshold: config.stages.execution.commit_threshold,
+                .set({
+                    let mut stage: ExecutionStage<'_, DefaultDB<'_>> =
```

Comment:
This `DefaultDB` thing is the only thing I don't like. It's necessary because when you instantiate the stage it still doesn't know the type of the database we'll use. It'd be nice if we can avoid it.

Reply:
It should be possible to avoid this if we get a `SubState` type that expires part of the cache (in a correct way) since we would be able to instantiate that once as well. For now this has to do.